modules/dynamic/src/smithy4s/dynamic/internals/DynamicModelCompiler.scala

```diff
@@ -18,17 +18,21 @@ package smithy4s
 package dynamic
 package internals
 
-import smithy4s.dynamic.model._
-
-import scala.collection.mutable.{Map => MMap}
-import smithy4s.schema.Schema._
-import smithy4s.internals.InputOutput
 import cats.Eval
 import cats.syntax.all._
-import smithy4s.schema.{Alt, EnumTag, EnumValue, Field}
+import smithy4s.dynamic.model._
```

Comment:
I think this `DynamicModelCompiler` will need to be updated to handle the new primitives similar to how it is handling UUID today. We'll want to be sure to add test cases around this too to make sure it is working properly (from a quick look I don't think we have any tests for UUID specifically here either so we can add that too). Anyway, lmk if you want to pair on this and I'll be happy to help